soh/src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -10872,8 +10891,15 @@ void Player_UpdateCamAndSeqModes(PlayState* play, Player* this) {
             } else if (this->stateFlags1 & PLAYER_STATE1_CHARGING_SPIN_ATTACK) {
                 camMode = CAM_MODE_CHARGE;
             } else if (this->stateFlags1 & PLAYER_STATE1_THREW_BOOMERANG) {
-                camMode = CAM_MODE_FOLLOWBOOMERANG;
-                Camera_SetParam(Play_GetCamera(play, 0), 8, this->boomerangActor);
+                // #region SOH [Enhancement]
+                if (CVarGetInteger(CVAR_ENHANCEMENT("BoomerangFirstPerson"), 0)) {
+                    camMode = CAM_MODE_TARGET;
+                    Camera_SetParam(Play_GetCamera(play, 0), 8, this->boomerangActor);
+                // #endregion
+                } else {
+                    camMode = CAM_MODE_FOLLOWBOOMERANG;
+                    Camera_SetParam(Play_GetCamera(play, 0), 8, this->boomerangActor);
+                }
```

Comment:
Can we move the `Camera_SetParam(Play_GetCamera(play, 0), 8, this->boomerangActor);` out of this if block